supervisor/core.py

```diff
@@ -150,7 +150,7 @@ async def start(self):
             _LOGGER.warning("System running in a unsupported environment!")
         if not self.healthy:
             _LOGGER.critical(
-                "System running in a unhealthy state and need manual intervention!"
+                "System running in a unhealthy state and will require manual intervention!"
```

Comment:
Besides it missing an `s`, there was no need to change it into `will require`.
There was, however, an `is` missing and it's `an unhealthy` 😉 

```suggestion
                "System is running in an unhealthy state and needs manual intervention!"
```

Reply:
Yep, makes more sense